### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by Clexus**
修改了一些语法错误和加了一些小细节
Fixed some grammatical errors and add some details

### CODE REVIEW

**Comment by Clexus** [comment]
can I be the proofreader?

**Comment by Alexander01998** [comment]
> can I be the proofreader?

You cannot be the proofreader of your own translation. You can only proofread other people's translations.

File: src/main/resources/assets/wurst/lang/zh_cn.json [hunk 1]
```diff
@@ -70,9 +70,9 @@
   "description.wurst.hack.instantbunker": "建造一个小小的火柴盒，需要57个方块。",
   "description.wurst.hack.itemesp": "高亮附近的掉落物品。",
   "description.wurst.hack.itemgenerator": "生成随机的物品并将其丢在地上。\n§o仅限创造模式。§r",
-  "description.wurst.hack.jesus": "允许你走在水面上。\n耶稣曾经用这个功能约 ~2000 年前。",
-  "description.wurst.hack.jetpack": "模仿你好像真的有飞行背包一样飞行。\n\n§c§l警告：§r 如果你不使用 NoFall，会导致受到掉落伤害。",
-  "description.wurst.hack.kaboom": "像爆炸一样破坏你附近的方块。\n这个可能比 Nuker 功能破坏更快如果服务器没有反作弊的话，使用适合的工具和破坏脆弱的方块可以事半功倍。\n注意：这并不是真正的爆炸。",
+  "description.wurst.hack.jesus": "允许你走在水面上。\n耶稣曾经在约 2000 年前就用过这个功能了。",
+  "description.wurst.hack.jetpack": "让你好像真的有飞行背包一样飞行。\n\n§c§l警告：§r 如果你不使用 NoFall，会导致受到掉落伤害。",
+  "description.wurst.hack.kaboom": "像爆炸一样破坏你附近的方块。\n这个可能比 Nuker 功能破坏更快，如果服务器没有反作弊的话，使用适合的工具和破坏脆弱的方块可以事半功倍。\n注意：这并不是真正的爆炸。",
```
Comment (xtexChooser) [inline src/main/resources/assets/wurst/lang/zh_cn.json:75]: 条件放前面好点吧（

File: src/main/resources/assets/wurst/lang/zh_cn.json [hunk 2]
```diff
@@ -87,22 +87,22 @@
...
-  "description.wurst.hack.nohurtcam": "关闭因受伤而产生视野摇晃效果。",
-  "description.wurst.hack.nooverlay": "防止因在水中或岩浆产生视野阻挡。",
+  "description.wurst.hack.nofall": "使你从免受摔落伤害。",
+  "description.wurst.hack.nofireoverlay": "防止火焰阻挡视野。\n\n§c§l警告：§r这可能会导致你因不知道身上有火而被烧死。",
+  "description.wurst.hack.nohurtcam": "关闭因受伤而产生的视野摇晃效果。",
+  "description.wurst.hack.nooverlay": "防止因在水中或岩浆中产生的视野阻挡。",
   "description.wurst.hack.nopumpkin": "防止因带南瓜头而导致视野阻挡。",
-  "description.wurst.hack.noslowdown": "取消减速的效果。像蜂蜜、灵魂沙或者使用物品。",
-  "description.wurst.hack.noweather": "允许改变你当地客户端所在的时间，和月亮的阶段（仅在本地生效，与服务器无关）。",
+  "description.wurst.hack.noslowdown": "取消减速的效果。像蜂蜜、灵魂沙或者使用物品时。",
```
Comment (xtexChooser) [inline src/main/resources/assets/wurst/lang/zh_cn.json:97]: “蜂蜜、灵魂沙”建议加个动词

File: src/main/resources/assets/wurst/lang/zh_cn.json [hunk 3]
```diff
@@ -116,33 +116,33 @@
...
+  "description.wurst.hack.servercrasher": "生成一个可以崩掉 1.15.x 的服务器的物品。\n§o仅限创造模式§r。",
   "description.wurst.hack.skinderp": "随机切换你的皮肤。",
   "description.wurst.hack.sneak": "让你自动保持潜行行走。",
   "description.wurst.hack.snowshoe": "可以让你在细雪上行走。",
   "description.wurst.hack.speedhack": "允许你以大约 ~2.5x 或更快的速度跳跃或疾跑。\n\n§6§l警告：§r只有低于反作弊版本 3.13.2 才会工作且绕过，如果你不知道服务器反作弊是哪个版本。\n输入 §l/ncp version§r 来查看反作弊版本。",
   "description.wurst.hack.speednuker": "一个比 Nuker 更快的版本，但会无法绕过反作弊。",
   "description.wurst.hack.spider": "像蜘蛛一样能够爬墙。",
   "description.wurst.hack.step": "允许你走上更高高度（取决于你的设定）的格子方块。",
-  "description.wurst.hack.throw": "迅速丢你手中种类的物品，可以用于丢雪球、鸡蛋、刷怪蛋、矿车，诸如此类。\n\n会生成大量的数量，这会产生大量的 lag 且可能会导致服务器崩溃。",
+  "description.wurst.hack.throw": "迅速丢出你手中种类的物品，可以用于丢雪球、鸡蛋、刷怪蛋、矿车，诸如此类。\n\n会生成大量的该种实体，这会产生大幅的卡顿且可能会导致服务器崩溃。",
```
Comment (xtexChooser) [inline src/main/resources/assets/wurst/lang/zh_cn.json:127]: “手中种类的物品”？

File: src/main/resources/assets/wurst/lang/zh_cn.json [hunk 4]
```diff
@@ -116,33 +116,33 @@
...
+  "description.wurst.hack.tired": "让你看起来很像 Alexander，回到2015年4月。\n仅限其他玩家可视。",
   "description.wurst.hack.toomanyhax": "屏蔽那些你并不想要有的功能。\n让你确保你不会不小心开启错误的作弊功能导致被服务器封禁。\n这个是给那些 “只想作弊一点点的用户”。\n\n使用指令 §6.toomanyhax§r 来选择哪些功能需要被屏蔽。\n输入 §6.help toomanyhax§r 寻求更多。",
   "description.wurst.hack.tp-aura": "自动传送到实体附近并自动攻击他。",
   "description.wurst.hack.trajectories": "预测箭和投掷物品的飞行路径。",
   "description.wurst.hack.treebot": "一个实验性的机器人将会在树木附近砍树。\n目前仅限于砍小树。",
   "description.wurst.hack.triggerbot": "自动攻击你所看的实体。",
   "description.wurst.hack.trollpotion": "生成一个惹人恼怒的药水效果。\n§o仅限创造模式。§r",
   "description.wurst.hack.truesight": "允许你看到隐身的实体。",
-  "description.wurst.hack.tunneller": "自动挖一个隧道。\n\n§c§l警告：§r 尽管这个 AI 会自动避开岩浆和其他危险的地下，这里没有守卫确保你不会死亡，除非你不介意你的东西全部有一定概率会失去。",
+  "description.wurst.hack.tunneller": "自动挖一个隧道。\n\n§c§l警告：§r 尽管这个 AI 会自动避开岩浆和其他危险的地方，但这里没有守卫确保你不会死亡，除非你不介意你所有的东西都有可能因此失去。",
```
Comment (xtexChooser) [inline src/main/resources/assets/wurst/lang/zh_cn.json:138]: “会自动”应该是“会尝试”（”will try to“）
“守卫”应该是“保证”（“guarantee”）

**Review by xtexChooser — CHANGES_REQUESTED** [review]
Good Job

File: src/main/resources/assets/wurst/lang/zh_cn.json (hunk 3 above)
Comment (buiawpkgew1) [reply src/main/resources/assets/wurst/lang/zh_cn.json:127]: "主手中" ?

Comment (xtexChooser) [reply src/main/resources/assets/wurst/lang/zh_cn.json:127]: 原文是多次使用同一种物品，不一定是丢，但是确实用的是主手

**Comment by xtexChooser** [comment]
cc @Clexus

**Comment by Alexander01998** [comment]
@xtexChooser This pull request was abandoned in April and many of its translations now conflict with the master branch. I don't think this will get merged anymore...

File: src/main/resources/assets/wurst/lang/zh_cn.json (hunk 4 above)
Comment (Clexus) [reply src/main/resources/assets/wurst/lang/zh_cn.json:138]: 我直接按照错误翻译来的，没看原文

Comment (Clexus) [reply src/main/resources/assets/wurst/lang/zh_cn.json:138]: > “会自动”应该是“会尝试”（”will try to“） “守卫”应该是“保证”（“guarantee”）

x

**Pull request closed without merging**